src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,98 @@
 package core.basesyntax;
 
-/**
- * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
- * реалізації ArrayList (default capacity, newCapacity...)</p>
- */
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEF_CAPACITY = 10;
+    private Object[] element;
+    private int capacity;
+    private int size;
+
+    public ArrayList() {
+        element = new Object[DEF_CAPACITY];
+    }
+
+    private void resize(int minCapacity) {
+        if (minCapacity > element.length) {
+            int newCapacity = (minCapacity * 3) / 2 + 1;
```

Comment:
```suggestion
            int newCapacity = (element.length * 3) / 2 + 1;
```
we should increase the capacity, right?

Reply:
Agry